python-avd/pyavd/_eos_designs/schema/schema_fragments/defs_node_type.schema.yml

```diff
@@ -405,7 +409,6 @@ $defs:
               - level-1-2
               - level-1
               - level-2
-            default: level-2
```

Comment:
for this one I would add a description that it overrides the main `isis_default_is_type`


```
        description: Overrides `isis_default_is_type`
```